lib/vanilla.js

```diff
@@ -0,0 +1,10 @@
+(function(context) {
+    var http = new VanillaAdapter(context.XMLHttpRequest, context.Promise);
+
+    context.taggedApi = new TaggedApi('/api/', {
```

Comment:
We should expose the constructor instead of creating an instance because we may want to provide custom constructor arguments.